build.py

```diff
@@ -1643,8 +1643,15 @@ def create_build_dockerfiles(
 ):
     if "base" in images:
         base_image = images["base"]
+        if target_platform() == "rhel":
+            print(
+                "warning: RHEL is not an officially supported target and you will probably experience errors attempting to build this container."
+            )
     elif target_platform() == "windows":
         base_image = "mcr.microsoft.com/dotnet/framework/sdk:4.8"
+    elif target_platform() == "rhel":
+        # Disallow RHEL target unless base image is specified
```

Comment:
This comment is redundant.

Reply:
I mean if you think the exception notice is not informative enough may be it better to add one more sentence?

Reply:
```suggestion
```